cc: error: unrecognized command line option '-NDEBUG'

checking for OS
 + Linux 4.4.0-93-generic x86_64
checking for C compiler ... found
 + using GNU C compiler
 + gcc version: 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.5) 


when running "./configure" with "non-Y" for "Do you want to enable debug features (not recommended for production usage) [y/N]: "
got below error:
./configure: error: can not detect int size


from ${NGINX_SRC_DIR}/objs/autoconf.err
cc: error: unrecognized command line option '-NDEBUG'

--- a/configure
+++ b/configure
@@ -30,7 +30,7 @@ fi
 if [[ "$NGINX_DEBUG" =~ ^([yY][eE][sS]|[yY])+$ ]]; then
     CONFIG_OPTS=(${CONFIG_OPTS[@]} --with-debug --with-cc-opt="-O0 -g")
 else
-    CONFIG_OPTS=(${CONFIG_OPTS[@]} --with-cc-opt="-NDEBUG")
+    CONFIG_OPTS=(${CONFIG_OPTS[@]} --with-cc-opt="-DNDEBUG")
 fi
 
 BUILD_INFO=("NGINX_SRC_DIR=$NGINX_SRC_DIR" "NGINX_VERSION=$NGINX_VERSION" "NGINX_DEBUG=$NGINX_DEBUG")

